storybook/src/components/Card/Card.stories.tsx

```diff
@@ -37,9 +37,9 @@ type Story = StoryObj<typeof meta>
 export const Default: Story = {
   args: {
     children: [
-      <Heading key={1} level={1} size="level-4">
+      <Card.Heading key={1} level={1} size="level-4">
```

Comment:
We don't need to set the `size` in these stories if we set a default in CardHeading